app/models/pair_request.rb

```diff
@@ -2,17 +2,29 @@
 #
 # Table name: pair_requests
 #
-#  id          :bigint           not null, primary key
-#  duration    :float
-#  when        :datetime
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
-#  acceptor_id :integer
-#  author_id   :integer
+#  id         :bigint           not null, primary key
+#  duration   :float            not null
+#  status     :integer          default(0), not null
+#  when       :datetime         not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  author_id  :integer          not null
+#  invitee_id :integer          not null
 #
 class PairRequest < ApplicationRecord
   belongs_to :author, class_name: "User"
-  belongs_to :acceptor, class_name: "User", optional: true
+  belongs_to :invitee, class_name: "User"
 
-  validates :when, presence: true
+  validates :when,
+            presence: true,
+            inclusion: { in: (Date.today..(Date.today + 1.month)) }
+  validates :duration, presence: true, numericality: { greater_than_or_equal_to: 900.0 }
+  validates :status, presence: true
+
```

Comment:
We should prob add a `before_create` hook to set this as pending always.

Reply:
hmm, this is on top of the default value in the migration?

Reply:
ignore this, I feel kinda old because it used to have issues. Yeah when you do `PairRequest.new` it will set the defaults for you.